tests/js/common/shell/shell-validation-rocksdb.js

```diff
@@ -360,6 +360,28 @@ function ValidationBasicsSuite () {
       }
 
     },
+    // AQL  ////////////////////////////////////////////////////////////////////////////////////////////
+    testGET_SCHEMA: () => {
+      const v =  validatorJson;
+      v.level = "strict";
```

Comment:
Doesn't this modify the original variable `validatorJson`? Is that expected behavior, because the same variable is used in the follow-up tests?

Reply:
In the setup every `validatorJSON` is reset for each test so it is safe to modify.

Reply:
ok, got it.